tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/PartialInlinePassImpl.scala

```diff
@@ -0,0 +1,91 @@
+package at.forsyte.apalache.tla.pp.passes
+
+import at.forsyte.apalache.infra.passes.PassOptions
+import at.forsyte.apalache.io.lir.{TlaWriter, TlaWriterFactory}
+import at.forsyte.apalache.tla.imp.findBodyOf
+import at.forsyte.apalache.tla.lir.storage.{BodyMap, BodyMapFactory}
+import at.forsyte.apalache.tla.lir.transformations._
+import at.forsyte.apalache.tla.lir.transformations.standard.ModuleByExTransformer
+import at.forsyte.apalache.tla.lir.{ModuleProperty, TlaModule, TlaOperDecl}
+import at.forsyte.apalache.tla.pp._
+import com.typesafe.scalalogging.LazyLogging
+
+/**
+ * A pass that expands operators and let-in definitions.
+ *
+ * @param options
+ *   pass options
+ * @param gen
+ *   name generator
+ * @param tracker
+ *   transformation tracker
+ * @param nextPass
+ *   next pass to call
```

Comment:
I don't think these documentation on the variables are helpful. I'd suggest either making them informative or removing them.

Reply:
I didn't write this, this entire class is just `InlinePassImpl` on unstable with the transformation sequence cut out. But I can axe them if you'd like.

Reply:
Ah yes. I realized that you didn't add this code later but forgot to update.